src/resty/gcp/init.lua

```diff
@@ -51,50 +51,93 @@ FindApis = function(apiClass, methods, curr)
     return methods
 end
 
+local function template_expansion(str, params)
+    return (string.gsub(str, "{(.-)}", function (key)
+        return ngx.escape_uri(params[key])
+    end))
+end
+
+local function build_request(accesstoken, apiDetail, params, requestBody, opt)
+    local req = {
+        method = apiDetail.httpMethod,
+        headers = {
+            ["Authorization"] = "Bearer " .. accesstoken.token
+        },
+        body = requestBody,
+        ssl_verify = false
```

Comment:
Any reason to turn it off by default?

Reply:
IDK. This is copied from original code (it's a refactoring).

Reply:
Captured this in an issue (#10) for now.